tests/integ/test_udf.py

```diff
@@ -1211,14 +1211,18 @@ def test_add_packages_negative(session, caplog):
 
     with pytest.raises(ValueError) as ex_info:
         with caplog.at_level(logging.WARNING):
-            session.add_packages("numpy", "numpy==0.1.0")
+            session.add_packages("numpy", "numpy==1.16.6")
```

Comment:
here 1.16.6 is the numpy version that is supported on the server side and not installed on the client side, right? I feel it's not good since it might not be supported on the server side in the future, and we can't control the version installed on the client side, but I couldn't think of a better solution and since it's just a test, maybe add a comment here to explain why do you use 1.16.6?

Reply:
handled in the latest commit